VibeSomeMauiBro.UITests/BaseTest.cs

```diff
@@ -0,0 +1,122 @@
+using OpenQA.Selenium.Appium;
+using OpenQA.Selenium.Appium.Android;
+using System.Diagnostics;
+
+namespace VibeSomeMauiBro.UITests;
+
+public abstract class BaseTest : IDisposable
+{
+    protected AndroidDriver Driver { get; private set; } = null!;
+    
+    public string PackageName { get; } = "com.companyname.vibesomemauibro";
+    public string ActivityName { get; } = "com.companyname.vibesomemauibro.MainActivity";
+    
+    private static readonly string ArtifactsPath = Path.Combine(Environment.CurrentDirectory, "test-artifacts");
```

Comment:
There are no `tests-artifacts` files from these tests!

`Environment.CurrentDirectory` won't work here.

If you do:
```csharp
string assemblyDirectory = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)
```

That will be a path like `./VibeSomeMauiBro.UITests/bin/Debug/net9.0/`.

So, you'll need to do something like `Path.Combine(assemblyDirectory, "..", "..", "..", "test-artifacts")`.

Add some logging, so we'll know if something goes wrong.